### PR TITLE
Add the possibility to set a custom domain

### PR DESCRIPTION
**Pull request by PatrikElfstrom**
# Pull Request Details

## Related Issue
#68 

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [X] My code follows the code style of this project.
- [X] I have updated the documentation or my changes dont require it.
- [X] I have read the [CONTRIBUTING](CONTRIBUTING.md) document.
- [X] I have added tests to cover my changes (or the existing one were enough).
- [X] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ serve ~/myproj
 - `sudo` may be necessary.
 - If a static path is not provided the current directory content will be served.
 - You can change the **port** setting the `PORT` environmental variable: `PORT=4433 serve ~/myproj`. Specifying port number will also prevent http to https redirect.
+- You can change the **domain** setting the `DOMAIN` environmental variable: `DOMAIN=192.168.0.2 serve ~/myproj`.
```
Comment (daquinoaldo) [inline README.md:46]: The variable is called DOMAIN but you are passing an IP address. What about calling it host or change the example?

**Review by daquinoaldo — COMMENTED** [review]
Amazing, thank you very much! Just a minor comment and we can merge it soon! 🙌

File: README.md (hunk 1 above)
Comment (PatrikElfstrom) [reply README.md:46]: Yes your right. Host is probably a better naming.

**Review by daquinoaldo — APPROVED** [review]
I'll release it in a few days. Thank you for contributing! 🙌

**Comment by daquinoaldo** [comment]
Released v4.7.0

**Pull request merged**